src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -189,7 +189,10 @@ protected function describeContainerDefinition(Definition $definition, array $op
         $output .= "\n".'- Abstract: '.($definition->isAbstract() ? 'yes' : 'no');
 
         if (method_exists($definition, 'isAutowired')) {
```

Comment:
Probably that FrameworkBundle wasn't requiring DI v2.8 when I started working on autowiring. I'll double check (and open a PR if you want me to) but I think that it can be removed now.